lua/telescope/_extensions/file_browser/make_entry.lua

```diff
@@ -159,6 +161,10 @@ local make_entry = function(opts)
       end
     end
 
+    if is_symlink then
+        path_display = string.format("%s -> %s ",path_display,entry.readlink);
```

Comment:
We should probably use `transform_path` on the real path as well
```suggestion
        path_display = string.format("%s -> %s", path_display, utils.transform_path(opts, entry.realpath));
```